setup.py

```diff
@@ -45,42 +45,18 @@
 ###############################################################################
 # Key Values that Change Each Release
 ###############################################################################
-SETUP_REQUIREMENTS = {
-    "numpy": "1.17",  # released July 2019
-    "scipy": "1.3",  # released May 2019
+# These are strictly installation requirements. Builds requirements are
+# managed in pyproject.toml
+INSTALL_REQUIREMENTS = {
+    "numpy": "1.18",  # released December 2019
```

Comment:
@bashtage We were having issues using statsmodels with Python 3.9 on a Red Hat cluster due to the install requirement on numpy 1.17 (see https://github.com/numpy/numpy/issues/14147).    Glad to see this has been upgraded to numpy 1.18.  Any plans for a pypi release in the near future that would include this change?

Reply:
Can you not set the c99 flag and build with that in the environment?  I'll see if I can cut a 0.13.2 this week.  It is overdue a bit.

Reply:
Normally we would not bump install req for a released version.  These changes are for the 0.14 release.

Reply:
Thanks for the quick response!  We can use the c99 flag, but our environment makes it difficult to do so cleanly.  If this change is not coming until 0.14, we may delay using statsmodels in combination with Python 3.9 until then.

Reply:
I suspect that it won't be out until 2014, although the build requirements will change - it is possible that the build reqs are the main issue rather than the minimum install reqs. The build reqs for 3.7 will likely drop to 1.14 since we have moved to oldest supported NumPy.